lops/lop-domain-a9-zynq-rpmsg-kernelspace.dts

```diff
@@ -56,7 +53,7 @@
                   };
                   lop_14 {
                          compatible = "system-device-tree-v1,lop,output";
-                         outfile = "linux.dts";
+                         outfile = "linux.dtb";
```

Comment:
Is the .dts not compiling after the operations are run ? Just curious, since this isn't mentioned in the commit message.

Reply:
it does. just trying to match convention of other lop files. that being said this commit can be cherry picked out if you wish